contract/asc/types.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="near-runtime-ts/assembly/as_types" />
```

Comment:
Why do we need this file in source?

Reply:
It adds the global types definitions of the runtime, e.g., `obj.JSON`, `decode<T>()`.  This could go in the tsconfig.json, but it currently points to asc's which limits the types to just the stdlib.